config/locales/pages/cookie_policy/en.yml

```diff
@@ -76,7 +76,7 @@ en:
         You’ve <span class="decision accepted">accepted</span><span class="decision rejected"> rejected</span> analytics cookies.
         You can %{cookie_preferences_link} at any time.
       page_link_text: change your cookie settings
-      button_text: Hide this message
+      button_text: Hide this <span class="govuk-visually-hidden">cookies</span> message
```

Comment:
```suggestion
      button_text_html: Hide this <span class="govuk-visually-hidden">cookies</span> message
```
This needs to be suffixed _html else it doesn't render html (and also [here](https://github.com/DFE-Digital/register-trainee-teachers/blob/9b1f3961754dcfba9652f1ab7b5c420c8c41fbf2/app/views/layouts/_cookie_banner.html.erb#L30))

![image](https://user-images.githubusercontent.com/910019/162223697-c1f65e82-f5a7-460d-9f13-d90b92551920.png)

Reply:
Thanks @archferns! — I can't get the app to run locally, so need to deploy and hope.